packages/qwik/src/testing/document.ts

```diff
@@ -20,53 +20,54 @@ export function createDocument(opts?: MockDocumentOptions) {
  * @public
  */
 export function createWindow(opts: MockDocumentOptions = {}): MockWindow {
-  const win = createDocument(opts).defaultView!;
-  return win;
+  return createDocument(opts).defaultView!;
 }
 
 export function ensureGlobals(doc: any, opts?: MockDocumentOptions) {
-  if (!doc[QWIK_DOC]) {
-    if (!doc || doc.nodeType !== 9) {
-      throw new Error(`Invalid document`);
-    }
+  if (!doc || doc.nodeType !== 9) {
```

Comment:
We are changing the behaviour here. If doc[QWIK_DOC] we need to return doc.defaultView